apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoader.java

```diff
@@ -480,7 +445,7 @@ private List<RifRecordLoadResult> process(
                     rifRecordEvent.getFileEvent().getFile().getFileType().name(),
                     rifRecordEvent.getRecordAction(),
                     ((Beneficiary) record).getBeneficiaryId(),
-                    rifData.toString());
+                    "");
```

Comment:
Oh, thats what we were using this for. When we skipped records we were saving them in this table to one day.. do something with. I don't think we ever came up with what to do with them though, and we theoretically shouldnt be getting any more skipped records in the future, so we're probably ok to proceed with this PR. However, without the record itself the table loses some of its purpose. We should think about if we still want this table and if we should still be recording these records; maybe a question for @mjburling

Reply:
Yeah @mackec1 and I came to the same conclusion.  464-474 should be restored and the `rifData` should continue to be included in the `SkippedRecord`.